### PR TITLE
fix: Combobox not showing options upon input click

### PR DESCRIPTION
**Pull request by julesglad**
UI fix for issue #2545

### CODE REVIEW

**Comment by GeorgeMac** [comment]
https://github.com/flipt-io/flipt/assets/1253326/cec591aa-588e-405f-83c2-7746e3dd6679

Hey @julesglad ! Thanks for taking this on.

I just gave it a build locally to try it out. The popover nicely populates for me now 🙌. However, I don't appear to be able to actually select any of the items now 🤔 I might try checkout and build main again to see if this problem is isolated to your change or not.

**Comment by julesglad** [comment]
@GeorgeMac I was able to recreate on my end. Found a fix and pushed a commit, and it's ready to be tested. The commit automatically attached to this pull request since it was on the same branch, I should need no further action, correct?

File: ui/src/components/forms/Combobox.tsx [hunk 1]
```diff
@@ -33,11 +33,17 @@ export default function Combobox<T extends IFilterable>(
...
 
   const filteredValues = values?.filter((v) =>
     v.filterValue.toLowerCase().includes(query.toLowerCase())
   );
 
+  console.log(query);
+  console.log('values filtered', filteredValues);
+  console.log(selected);
+  console.log(setSelected, 'set selected fired');
+
```
Comment (GeorgeMac) [inline ui/src/components/forms/Combobox.tsx:46]: These can probably go now?
```suggestion
```

**Review by GeorgeMac — CHANGES_REQUESTED** [review]
It works! Thanks @julesglad 🙏 There is one thing, just some debug statements need removing. Otherwise, I think it is good to go 🙌

**Comment by julesglad** [comment]
@GeorgeMac Oops, sorry about that. Removal pushed up now. 😳🫣🫣

**Comment by markphelps** [comment]
looks and works great! thanks @julesglad !!

**Pull request merged**